roles/docker/handlers/main.yml

```diff
@@ -4,10 +4,17 @@
   notify:
     - Docker | reload systemd
     - Docker | reload docker
-    - Docker | pause while Docker restarts
     - Docker | wait for docker
 
+# NOTE(bogdando) reloading systemd daemon disturbs docker, we shall wait for it
 - name : Docker | reload systemd
+  command: /bin/true
+  notify:
+    - Docker | systemd-daemon-reload
+    - Docker | wait for docker
```

Comment:
AFAICT, there is always
```
    - { role: docker, tags: docker }
```
assigned to a group members before we try to reload daemon. Preinstall tasks has none of such tasks, right?